test/load/terraform/variables.tf

```diff
@@ -111,3 +111,15 @@ variable "ami_owners" {
   type        = list(string)
   description = "The account owner number which the desired AMI is in"
 }
+
+variable "consul_download_url" {
+  type        = string
+  description = "URL to download the Consul binary from"
+}
```

Comment:
This will need to be set as an empty string ("") by default, since it's meant to be optional.